lib/improver/nbhood.py

```diff
@@ -290,15 +365,19 @@ def process(self, cube):
         else:
             if len(realiz_coord.points) > 1:
                 raise ValueError("Does not operate across realizations.")
+            else:
+                for cube_slice in cube.slices_over("realization"):
```

Comment:
Why slice over realization if you are enforcing only one point on that coordinate?